databaseutil/databaseutil.go

```diff
@@ -69,3 +69,21 @@ func GetPasswordForUserWithEmailAddress(emailAddress string) ([]byte, error) {
 
 	return password, nil
 }
+
+func GetUserIdFromUserWithEmailAddress(emailAddress string) (int64, error) {
```

Comment:
I thought database util was abstracting away the SQL from the function. Since UserId isn't a type in SQL, i thought it would make a better interface if it returned int64 the function that calls this one immediately casts int64 into UserId

Reply:
fair enough